core/views.py

```diff
@@ -25,6 +28,38 @@ def get_object(self: "Post", queryset: QuerySet[Post] | None = None) -> Post:
         obj.update_display_counter()
         return obj
 
+    def get_context_data(self: "PostDetailView", **kwargs: dict[str, Any]) -> dict[str, Any]:
+        context = super().get_context_data(**kwargs)
+        comments = self.object.get_comments()
+        form = CommentForm()
+
+        context["comments"] = comments
+        context["form"] = form
+        return context
+
+    def post(self: "PostDetailView", request: HttpRequest, pk: int) -> HttpResponse:
+        post = get_object_or_404(Post, id=pk)
+        form = CommentForm(request.POST)
+        if form.is_valid():
+            new_comment = form.save(commit=False)
+            parent_id = request.POST.get("parent_id")
```

Comment:
This should be in the form, like so
```
from django import forms
from .models import Post

class PostForm(forms.ModelForm):
    extra_field = forms.CharField(max_length=100, required=False)

    class Meta:
        model = Post
        fields = ['title', 'content']  # Specyfikacja pól modelu
```

Reply:
created method for Post model that generate CommentForm, but changed CommentForm from forms.ModelForm to forms.Form, otherwise there would be import loop